src/homepage/images/james.svg

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="-223 283 164 228" style="enable-background:new -223 283 164 228;" xml:space="preserve">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
+     viewBox="-223 283 164 228" style="enable-background:new -223 283 164 228;" xml:space="preserve">
```

Comment:
did read the whole PR, but it seems very strange you have to modify this file

Reply:
Auto import optimization on whole project after moving Mailbox from store to api.
If not the checkstyle doesn't pass. It's in the same commit, if a change is  not desirable i can undo it.

Reply:
sorry, but all these changes are undesirable, or must at least be in a seperated commit to be easily reviewable.